### PR TITLE
chore: set `GITHUB_AUTH_TOKEN` for phive

### PR DESCRIPTION
**Pull request by datamweb**
ref: https://github.com/codeigniter4/CodeIgniter4/pull/7016
```console
Run sudo phive --no-progress install --global --trust-gpg-keys B8F640134AB1782E,A98E898BB53EB748 qossmic/deptrac
Phive 0.15.2 - Copyright (C) 2015-2023 by Arne Blankerts, Sebastian Heuer and Contributors
Downloading https://api.github.com/rate_limit
Warning:  Github API Rate Limit exceeded - cannot resolve "qossmic/deptrac"
Downloading https://gitlab.com/api/v4/projects/qossmic%2Fdeptrac/releases/
Fetching repository list
Downloading https://phar.io/data/repositories.xml
Error:    Could not resolve requested PHAR qossmic/deptrac

Error: Process completed with exit code 4.
```
see https://github.com/datamweb/shield/actions/runs/3862351738/jobs/6583866759

### CODE REVIEW

**Comment by datamweb** [comment]
It doesn't seem like the right solution to solve the problem. see:

https://github.com/codeigniter4/shield/actions/runs/3862455954/jobs/6584038136

File: .github/workflows/deptrac.yml [hunk 1]
```diff
@@ -70,3 +70,5 @@ jobs:
         run: |
           sudo phive --no-progress install --global --trust-gpg-keys B8F640134AB1782E,A98E898BB53EB748 qossmic/deptrac
           deptrac analyze --cache-file=build/deptrac.cache
+        env:
+          GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (kenjis) [inline .github/workflows/deptrac.yml:74]: Please add a new line.

**Comment by kenjis** [comment]
It is true that ”Github API Rate Limit exceeded” was reported even if `GITHUB_AUTH_TOKEN` is set.

https://github.com/codeigniter4/shield/actions/runs/3862455954/jobs/6584038136
```
Run sudo phive --no-progress install --global --trust-gpg-keys B8F640134AB1782E,A98E898BB53EB748 qossmic/deptrac
  sudo phive --no-progress install --global --trust-gpg-keys B8F640134AB1782E,A98E898BB53EB748 qossmic/deptrac
  deptrac analyze --cache-file=build/deptrac.cache
  shell: /usr/bin/bash -e {0}
  env:
    COMPOSER_PROCESS_TIMEOUT: 0
    COMPOSER_NO_INTERACTION: 1
    COMPOSER_NO_AUDIT: 1
    COMPOSER_AUTH: {"github-oauth": {"github.com": "***"}}
    COMPOSER_CACHE_FILES_DIR: /home/runner/.cache/composer/files
    GITHUB_AUTH_TOKEN: ***
Phive 0.15.2 - Copyright (C) 2015-2023 by Arne Blankerts, Sebastian Heuer and Contributors
Downloading https://api.github.com/rate_limit
Warning:  Github API Rate Limit exceeded - cannot resolve "qossmic/deptrac"
Downloading https://gitlab.com/api/v4/projects/qossmic%2Fdeptrac/releases/
Fetching repository list
Downloading https://phar.io/data/repositories.xml
Error:    Could not resolve requested PHAR qossmic/deptrac

Error: Process completed with exit code 4.
```

**Comment by kenjis** [comment]
Something wrong with this PR.
Screenshot 2023-01-09 8 55 33

**Comment by paulbalandan** [comment]
I can see the same. Maybe line 75 has extra whitespaces instead of an empty blank line.

**Comment by datamweb** [comment]
Looks like it's resolved.
But @Kenjis, I think we **shouldn't** merge this. My goal with these PR was to fix problem `deptract`, which I don't think can be fixed. As mentioned in this https://github.com/codeigniter4/CodeIgniter4/pull/7065#issuecomment-1375173599. Unless there is another reason for joining?
Do you agree with Close?

File: .github/workflows/deptrac.yml (hunk 1 above)
Comment (datamweb) [reply .github/workflows/deptrac.yml]: Done.

File: .github/workflows/deptrac.yml [hunk 2]
```diff
@@ -70,3 +70,6 @@ jobs:
...
+
```
Comment (kenjis) [inline .github/workflows/deptrac.yml:75]: We don't need the last empty line.

**Comment by kenjis** [comment]
@datamweb Yes, this does not seem to work.
I want to fix the GitHub rate limit issue, but now I don't have a clue.

**Comment by datamweb** [comment]
I close this PR, because now there is no problem for **Dependency Tracing**.

**Pull request closed without merging**